modules/administration/pages/kubernetes.adoc

```diff
@@ -8,10 +8,11 @@
 
 The prerequisites listed below should be met before proceeding.
 
-* At least one _Kubernetes_ or _SUSE CaaS Platform _ cluster available on your network
+* At least one _Kubernetes_ or _SUSE CaaS Platform_ cluster available on your network
 * {productname} configured for container management
 +
 
+// TODO: Is "note" the right choice here?
 [NOTE]
 ====
 Required channels are present, a registered container build host available, etc.
```

Comment:
This isn't a full sentence. I have no idea what it's trying to say.

Reply:
It's an explanation of the line above: "{productname} configured for container management"

Sorry, my TODO line makes it less obvious I guess

Reply:
OK, so are you explaining what "configured for container management" means? If that is the case, I would put that explanation in the dot point, not in an admonition.